symbolic-debuginfo/src/macho.rs

```diff
@@ -1,7 +1,7 @@
 //! Support for Mach Objects, used on macOS and iOS.
 
 use std::borrow::Cow;
-use std::error::Error;
+use std::error::Error as StdError;
```

Comment:
Is there a particular reason for this change? It would be best to keep `std::error::Error` without alias and qualify other errors that might conflict, instead.

Reply:
it's a fairly common pattern i've seen when using anyhow and/or thiserror.  but i don't mind much